app/src/main/res/values/strings.xml

```diff
@@ -2,4 +2,21 @@
     <string name="app_name">Accrescent</string>
     <string name="app_list">Apps</string>
     <string name="app_updates">Updates</string>
+    <string name="test">Just kidding. This is a shim.</string>
+    <string name="install">Install</string>
+    <string name="swipe_refresh">Swipe down to refresh</string>
+    <string name="user_action_required">User action required</string>
+    <string name="cant_find_app">Sorry, we couldn\'t find that app</string>
+    <string name="version">Version: %1$s</string>
+    <string name="version_code">Version code: %1$d</string>
+    <string name="device_language_unsupported">Your device\'s language is not supported</string>
+    <string name="device_density_unsupported">Your device\'s screen density is not supported</string>
+    <string name="device_abi_unsupported">Your device\'s ABI/architecture is not supported</string>
+    <string name="app_signer_mismatch">App not signed by required signer(s).\nExpected: %1$s</string>
+    <string name="no_app_signers">No app signers found.\nExpected: %1$s</string>
+    <string name="version_downgrade">Version %1$d is less than minimum %2$d</string>
+    <string name="app_id_mismatch">App ID %1$s does not match expected ID %2$s</string>
+    <string name="version_mismatch">Expected version %1$s, actual is %2$s</string>
+    <string name="version_code_mismatch">Expected version code %1$d, actual is %2$d</string>
```

Comment:
nit: Keep error messages starting with a lowercase letter. They're always displayed with context and not capitalizing them makes them easier to display in arbitrary places.